docs/content/advanced/admin/manage-publishable-api-keys.mdx

```diff
@@ -0,0 +1,525 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Manage Publishable API Keys
+
+In this document, you’ll learn how to manage the publishable API keys using the admin APIs.
+
+## Overview
+
+Publishable API keys allow you to associate a set of resources with an API key. You can then pass that API key in storefront requests to guarantee that processed or returned data is within the scope you defined when creating the API key.
+
+Currently, publishable API keys can only be associated with sales channels.
+
+Using the Admin APIs, you can manage your Publishable API Keys.
```

Comment:
suggestion: Add a note specifying that Publishable Keys are only for client-side use:
```suggestion
Using the Admin APIs, you can manage your Publishable API Keys.

:::note
Publishable API keys are only for client-side use. They can be public accessible in your code, as they are not authorized for the Admin API.
:::
```

Reply:
This would also fit in the conceptual guide and user guide.